os/drivers/i2c/i2c_ioctrl.c

```diff
@@ -0,0 +1,104 @@
+/****************************************************************************
+ * drivers/i2c/i2c_ioctrl.c
+ *
+ *   Copyright (C) 2016,2017 SAMSUNG ELECTRONICS CO., LTD. All rights reserved.
```

Comment:
Copyright is Samsung but license is nuttx. Let's remove this copyright and license and add Samsung Apach license.